requirements/system/user-client.tex

```diff
@@ -1,9 +1,291 @@
+      \begin{itemize}
+        \item Job proirity changes.
+        \item An appropriate success message is displayed.
+      \end{itemize}  
+      \item Post condition (fail):
+      \begin{itemize}
+        \item Invalid priority.
+        \item An appropriate error message is displayed.
+      \end{itemize}
+      \item Triggering Event: The user sets the priority of a job.
+      \item Description:
+      \begin{itemize}
+        \item The user adds the priority of the job while adding it to the queue with the "add command".
+        \item The user changes the priority of a job after it already start running with the "set ".
+      \end{itemize}
+    \end{itemize}
+
+
+    \item UCFR08 Checking the specifications of available containers
+    \begin{itemize}
+      \item Goal: User receives a description of the available containers.
+      \item Importance: Primary
+      \item Precondition: User must be logged in.
+      \item Post condition (success): A description of the available containers is displayed.
+      \item Post condition (fail):
+      \begin{itemize}
+        \item Networking error.
+        \item An appropriate error message is displayed.
+      \end{itemize}
+      \item Triggering Event: The user requests containers specifications.
+      \item Description:
+      \begin{itemize}
+        \item The user requests containers specifications with the "add command".
+      \end{itemize}
+    \end{itemize}
+
+    \item  UCFR09 Choosing the parameters of the container(preferred OS/environment)
+    \begin{itemize}
+      \item Goal: User chooses preffered container parameters for a specific job.
+      \item Importance: Primary
+      \item Precondition: User must be logged in.
+      \item Post condition (success): An appropriate success message is displayed.
+      \item Post condition (fail):
+      \begin{itemize}
+        \item unavailable specifications.
+        \item An appropriate error message is displayed.
+      \end{itemize}
+      \item Triggering Event: The user chooses containers environment.
+      \item Description:
+      \begin{itemize}
+        \item The user chooses containers environment in the "add command".
+      \end{itemize}
+    \end{itemize}
+
+    \item  UCFR10 Choosing the running mode
+    \begin{itemize}
+      \item Goal: User chooses preffered mode for the application to use.
+      \item Importance: Primary
+      \item Precondition: User must be logged in.
+      \item Post condition (success): An appropriate success message is displayed.
+      \item Post condition (fail):
+      \begin{itemize}
+        \item unavailable mode.
+        \item An appropriate error message is displayed.
+      \end{itemize}
+      \item Triggering Event: The user chooses a specific mode.
+      \item Description:
+      \begin{itemize}
+        \item The user chooses the application mode in the "add command".
+        \item The blocking mode lets the application run until the job is finished or stopped.
```

Comment:
Incorrect.
I suggest the following wording:
When the user chooses regular mode the user client will instantly return to the regular command line.
When the user chooses blocking mode the user client will only return to the command line once the job has finished running.